docs/content/cookbook/handling-custom-occ-endpoint.md

```diff
@@ -0,0 +1,182 @@
+# Handling custom OCC endpoints
+
+It is a common task to add support for a custom SAP OCC API endpoint. This guide will show you how can do it using Alokai.
+
+## Prerequisites
+
+Before we start make sure that you are familiar with Adding an Endpoint guide. With that guide, you would be able to
+communicate with OCC API but it would require manual retrieval of context parameters (baseSiteId, userId, language,
+and currency) and preparation of authorization headers. Read on to see how to streamline that process.
+
+## Communicating with OCC API effectively
```

Comment:
IMO, I think this section could use more background info on...

1. The overall process of using `client.get` like what is the client, is there an advantage to doing that over just a `fetch`, etc.
2. Where does this file go? I know you mentioned the Adding an API Endpoint, but having that information here too would be helpful.

Reply:
Good points! However, I won't be able to work on them before the workshops. I'll work on that next week.